analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/ConditionEvaluatesToConstant.CSharp9.cs

```diff
@@ -121,11 +121,11 @@ public object InitWithTupleAssignment
             {
                 var tmp = 0;
                 var flag = true;
-                while (flag)        // Noncompliant
+                while (flag)        // Compliant
```

Comment:
This was an FP, but wasn't marked as one earlier (see [this issue](https://github.com/SonarSource/sonar-dotnet/issues/3239)).